model/src/main/kotlin/licenses/LicenseView.kt

```diff
@@ -25,8 +25,8 @@ import org.ossreviewtoolkit.spdx.SpdxSingleLicenseExpression
 
 /**
  * A [LicenseView] provides a custom view on the licenses that belong to a [Package]. It can be used to filter the
- * licenses relevant to a [Rule] whereas the [licenseSources] is the filter criteria. Only the entry with the lowest
- * index in the given [licenseSources] which yields a non-empty result is used as filter criteria.
+ * licenses relevant to an evaluator rule whereas the [licenseSources] is the filter criteria. Only the entry with the
```

Comment:
As the *model*, like you say, does not know anything about an *evaluator*, should we drop that word here an just say "a rule" / "some rule" here, very similar to like it was before?

Reply:
I think it's fine to mention the evaluator here, we cannot link it because the code does not know about it, but the reader can know about it and it helps with understanding.